pkgs/desktops/gnome/core/eog/default.nix

```diff
@@ -38,6 +38,13 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-+zW/tRZ6QhIfWae5t6wNdbvQUXua/W2Rgx6E01c13fg=";
   };
 
+  patches = [
+    # Fix path to libeog.so in the gir file.
+    # We patch gobject-introspection to hardcode absolute paths but
+    # our Meson patch will only pass the info when install_dir is absolute as well.
```

Comment:
Ideally, we would change the Meson patch to always pass `--fallback-library-path` but we need to think about it more.